projects/cdk/schematics/ng-update/v4/steps/templates/migrate-axes.ts

```diff
@@ -0,0 +1,45 @@
+import type {UpdateRecorder} from '@angular-devkit/schematics';
+import type {DevkitFileSystem} from 'ng-morph';
+
+import {findElementsByTagName} from '../../../../utils/templates/elements';
+import {
+    getTemplateFromTemplateResource,
+    getTemplateOffset,
+} from '../../../../utils/templates/template-resource';
+import type {TemplateResource} from '../../../interfaces';
+import {removeAttrs} from '../utils/remove-attrs';
+
+const AXES_TAG_NAME = 'tui-axes';
+const AXES_NEW_ATTRIBUTE = 'new';
+
+export function migrateAxes({
+    resource,
+    recorder,
+    fileSystem,
+}: {
+    fileSystem: DevkitFileSystem;
+    recorder: UpdateRecorder;
+    resource: TemplateResource;
+}): void {
+    const template = getTemplateFromTemplateResource(resource, fileSystem);
+    const templateOffset = getTemplateOffset(resource);
+
+    const elements = findElementsByTagName(template, AXES_TAG_NAME);
+
+    elements.forEach(({attrs, sourceCodeLocation}) => {
+        const axesNewAttr = attrs.find((attr) => attr.name === AXES_NEW_ATTRIBUTE);
+
+        if (!sourceCodeLocation) {
+            return;
+        }
+
+        if (axesNewAttr) {
+            removeAttrs([axesNewAttr], sourceCodeLocation, recorder, templateOffset);
+        } else {
+            recorder.insertLeft(
+                sourceCodeLocation.startOffset + templateOffset,
+                '<!-- TODO: (Taiga UI migration) labels positioning on x axes were updated. Add empty string to axisXLabels array. See https://taiga-ui.dev/charts/axes -->\n',
```

Comment:
Let's extend this array 
https://github.com/taiga-family/taiga-ui/blob/78ef961d677fd8056599af2df03cbd1abb33c1e5/projects/cdk/schematics/ng-update/v4/steps/constants/html-comments.ts#L3

with the following object
```ts
{
    tag: 'tui-axes',
    filterFn: (el) => !hasElementAttribute(el, 'new'),
    comment:
        'labels positioning on x axes were updated. Add empty string to axisXLabels array. See https://taiga-ui.dev/charts/axes',
},
```

`filterFn` is not supported for this constant.
But feel free to add this feature.

Reply:
Unfortunately, I missed your comment and PR is already closed